default_bus_layout.tres

```diff
@@ -1,41 +1,59 @@
 [gd_resource type="AudioBusLayout" load_steps=4 format=2]
 
-[sub_resource type="AudioEffectLowPassFilter" id=1]
-resource_name = "LowPassFilter"
-cutoff_hz = 20500.0
-
 [sub_resource type="AudioEffectReverb" id=2]
 resource_name = "Reverb"
 room_size = 0.2
 wet = 0.0
 
+[sub_resource type="AudioEffectLowPassFilter" id=1]
+resource_name = "LowPassFilter"
+cutoff_hz = 20500.0
+
 [sub_resource type="AudioEffectEQ10" id=3]
 resource_name = "EQ10"
 band_db/8000_hz = -60.0
 band_db/16000_hz = -60.0
 
 [resource]
-bus/0/effect/0/effect = SubResource( 1 )
-bus/0/effect/0/enabled = true
-bus/0/effect/1/effect = SubResource( 2 )
-bus/0/effect/1/enabled = true
-bus/1/name = "Music"
+bus/1/name = "~~Water Muffle:"
 bus/1/solo = false
 bus/1/mute = false
 bus/1/bypass_fx = false
 bus/1/volume_db = 0.0
 bus/1/send = "Master"
-bus/2/name = "SFX"
+bus/1/effect/0/effect = SubResource( 2 )
+bus/1/effect/0/enabled = true
+bus/2/name = "~~Water Muffle:"
 bus/2/solo = false
 bus/2/mute = false
 bus/2/bypass_fx = false
 bus/2/volume_db = 0.0
-bus/2/send = "Master"
-bus/3/name = "Coin"
+bus/2/send = "~~Water Muffle:"
+bus/3/name = "SFX"
 bus/3/solo = false
 bus/3/mute = false
 bus/3/bypass_fx = false
 bus/3/volume_db = 0.0
-bus/3/send = "SFX"
-bus/3/effect/0/effect = SubResource( 3 )
-bus/3/effect/0/enabled = true
+bus/3/send = "~~Water Muffle:"
+bus/4/name = "Coin"
+bus/4/solo = false
+bus/4/mute = false
+bus/4/bypass_fx = false
+bus/4/volume_db = 0.0
+bus/4/send = "SFX"
+bus/4/effect/0/effect = SubResource( 3 )
+bus/4/effect/0/enabled = true
+bus/5/name = "~~SFX Clear In Water~~"
+bus/5/solo = false
+bus/5/mute = false
+bus/5/bypass_fx = false
+bus/5/volume_db = 0.0
+bus/5/send = "SFX"
+bus/5/effect/0/effect = SubResource( 3 )
+bus/5/effect/0/enabled = true
+bus/6/name = "~~SFX Clear In Water~~"
```

Comment:
Equally, why are there 2 buses named `~~SFX Clear In Water~~`?

And there is no bus named `~Water Verb:` for bus 6 to send to.